src/include/daos/mem.h

```diff
@@ -27,9 +27,24 @@
 
 int umempobj_settings_init(bool md_on_ssd);
 
+/* convert backend type to umem class id */
+int umempobj_backend_type2class_id(int backend);
+
 /* umem persistent object property flags */
 #define	UMEMPOBJ_ENABLE_STATS	0x1
 
+#ifdef DAOS_PMEM_BUILD
+enum {
+	DAOS_MD_PMEM	= 0,
+	DAOS_MD_BMEM	= 1,
+	DAOS_MD_ADMEM	= 2,
+};
+
+/* return umem backend type */
+int umempobj_get_backend_type();
```

Comment:
(style)  Bad function definition - int umempobj_get_backend_type() should probably be int umempobj_get_backend_type(void)